src/node/NodeWebSocketTransport.ts

```diff
@@ -20,7 +20,7 @@ import { packageVersion } from '../generated/version.js';
 export class NodeWebSocketTransport implements ConnectionTransport {
   static create(url: string): Promise<NodeWebSocketTransport> {
     return new Promise((resolve, reject) => {
-      const ws = new NodeWebSocket(url, [], {
+      const ws = new NodeWebSocket(url.replace('localhost', '127.0.0.1'), [], {
```

Comment:
I assume this is to workaround the DNS resolution issue? Note that this would break users on ipv6-only systems.

See also https://github.com/nodejs/node/issues/40537